swagger.yml

```diff
@@ -2239,34 +2239,38 @@ paths:
                     description: full paged
                     properties:
                       content:
-                        type: array
-                        items:
-                          $ref: '#/components/schemas/Database'
+                        type: object
+                        properties:
+                          databases:
+                            $ref: '#/components/schemas/Database'
                       metadata:
                         $ref: '#/components/schemas/PageMetadata'
                     required:
                       - content
                       - metadata
-                  - type: array
+                  - type: object
                     description: full non paged
-                    items:
-                      $ref: '#/components/schemas/Database'
+                    properties:
+                      databases:
+                        $ref: '#/components/schemas/Database'
                   - type: object
                     description: not full paged
                     properties:
                       content:
-                        type: array
-                        items:
-                          $ref: '#/components/schemas/Database'
+                        type: object
```

Comment:
Is this true?
It's still returned an array with the object database inside, not an object!